win32/scripts/regsetup.py

```diff
@@ -506,9 +506,9 @@ def RegisterShellInfo(searchPaths):
 as well as the 2 wierd spots to locate the core Python files (eg, Python.exe,
 pythonXX.dll, the standard library and Win32 Extensions.
 
-"regsetup -a myappname . .\subdir"
+"regsetup -a myappname . .\\subdir"
```

Comment:
This is an example of a command-line where the doubling is wrong too - perhaps the best fix here is to make that cleaner - eg, drop the quotes and add, say, `%` or `c:\>` as a prefix or similar?

Reply:
I don't mind if you suggest a replacement, or I can make this a raw-multiline-string and dropall the double slashes in this string from line 502-525

Reply:
yeah, as above, let's make sure it works both as a docstring *and* reads correctly in the editor.